From 5f25b3fba1b7a36edf12033e922d4e95047e15cd Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@armbian.com>
Date: Thu, 6 Feb 2025 10:21:13 -0500
Subject: [PATCH] HACK: meson64 boot target usb

By changing the USB boot order, it makes it possible
to recover the current install on the eMMC by booting
via USB.

NOTE: There is no SD Card support.

Signed-off-by: Patrick Yavitz <pyavitz@armbian.com>
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index f3275b37a51..3ace59880bd 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -119,8 +119,8 @@
 #define BOOT_TARGET_DEVICES(func) \
 	func(ROMUSB, romusb, na)  \
 	func(USB_DFU, usbdfu, na)  \
-	BOOT_TARGET_MMC(func) \
 	BOOT_TARGET_DEVICES_USB(func) \
+	BOOT_TARGET_MMC(func) \
 	BOOT_TARGET_NVME(func) \
 	BOOT_TARGET_SCSI(func) \
 	BOOT_TARGET_PXE(func) \
-- 
2.39.5

